Standards/scs-xxxx-v1-achieving-certification.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Regulations for achieving SCS-compatible certification
+type: Procedural
+status: Draft
+track: Global
+---
+
+## Introduction
+
+The Sovereign Cloud Stack (SCS) issues certificates with various scopes, among them _SCS-compatible IaaS_ (infrastructure as a service) and _SCS-compatible KaaS_ (Kubernetes as a service).
+
+This document details how a cloud service provider (henceforth also called operator) can attain such a certificate for one of their clouds.
+
+## Motivation
+
+As operator, I want to obtain a certificate with the scope SCS-compatible IaaS or SCS-compatible KaaS.
+
+## Regulations
+
+1. Each certificate issued pertains to a given cloud, a given scope, and a given version of that scope with a fixed expiry date. The certificate is only valid for that cloud and for the time frame that ends on that expiry date.
+
+2. The operator MUST include the official SCS compliance test suite (which does not require admin privileges) in their continuous test infrastructure (e.g., Zuul). The tests MUST be run at given intervals, depending on their classification:
+
+    - _light_: at least nightly,
+    - _medium_: at least weekly,
+    - _heavy_: at least monthly.
+
+   For public clouds, it is recommended to offer the SCS project access to the infrastructure (ideally, including a Zuul nodepool) so the test suite runs can be triggered continuously by the SCS team.
```

Comment:
I think the setup of a dedicated Zuul nodepool is not required to run the test suite. It should be sufficient to provide a project that can be accessed from the Zuul system.

Reply:
Added this question to the next meeting's agenda. https://input.scs.community/2023-scs-sig-standardization